browser/src/control/Control.Header.ts

```diff
@@ -543,6 +544,15 @@ export class Header extends CanvasSectionObject {
 		else { // We are in dragging mode.
 			this._dragDistance = dragDistance;
 			this.containerObject.requestReDraw(); // Remove previously drawn line and paint a new one.
+
+			if (this._lastSelectedIndex == this._mouseOverEntry.index)
+				return;
+			const modifier = this._lastSelectedIndex ? UNOModifier.SHIFT : 0;
+			this._lastSelectedIndex = this._mouseOverEntry.index;
+			if (this._isColumn)
```

Comment:
It's a bit weird that in base class we check type and manually choose method :)
It could be `selectIndex(index)` method which has different implementation in row and column classes...
Could you do that? To not make this pattern more popular here.